src/Symfony/Component/HttpFoundation/ParameterBag.php

```diff
@@ -155,6 +162,13 @@ public function getInt(string $key, int $default = 0): int
         return $this->filter($key, $default, \FILTER_VALIDATE_INT, ['flags' => \FILTER_REQUIRE_SCALAR]) ?: 0;
     }
 
+    public function getIntOrNull(string $key): ?int
+    {
+        return $this->has($key)
```

Comment:
Checking if the value is set is not necessary:
`filter_var(null, \FILTER_VALIDATE_INT, \FILTER_NULL_ON_FAILURE)` returns `null`.

Reply:
I've changed it to use `getInt` implementation instead.